tests/topotests/ospf_te_topo1/test_ospf_te_topo1.py

```diff
@@ -223,9 +223,6 @@ def test_step4():
     tgen.net["r2"].cmd(
         'vtysh -c "conf t" -c "router ospf" -c "segment-routing node-msd 16"'
     )
-    tgen.net["r2"].cmd(
-        'vtysh -c "conf t" -c "router ospf" -c "segment-routing local-block 5000 6999"'
-    )
```

Comment:
These lines should not be removed. Only line 227 should be replaced by:
`'vtysh -c "conf t" -c "router ospf" -c "segment-routing global-block 16000 23999 local-block 5000 6999"'`
As `local-block` CLI has been merged with `global-block` command, you should specified the default SRGB values to force SRLB values.